util-stats/src/test/java/com/twitter/finagle/stats/StatsReceiverCompilationTest.java

```diff
@@ -95,18 +100,18 @@ public boolean isNull() {
       }
 
       @Override
-      public Counter counter(Verbosity verbosity, Seq<String> name) {
+      public Counter counter(Verbosity verbosity, String... name) {
         return nullSr.counter(name);
       }
 
       @Override
-      public Stat stat(Verbosity verbosity, Seq<String> name) {
-        return nullSr.stat(name);
+      public Counter counter(Verbosity verbosity, scala.collection.Seq<String> name){
+        return nullSr.counter(verbosity, name.toVector());
       }
 
       @Override
-      public Gauge addGauge(Verbosity verbosity, Seq<String> name, Function0<Object> f) {
-        return nullSr.addGauge(verbosity, name, f);
+      public Gauge addGauge(Verbosity verbosity, scala.collection.Seq<String> name, Function0<Object> f) {
```

Comment:
I don't know how to fix this. I haven't found a way to have multiple parameter list with a varargs that is source-compatible for overriding between 2.13+ and 2.12-. If anyone has ideas, I'm happy to implement them, but for now, I'm out of ideas.

Reply:
implementing both without an override annotation works. This pattern can be further expanded: an abstract go between with both, one final forwarding to the abstract other.

Which one is better to finalize/forward is debatable.